docs/tutorials/on-device-training.md

```diff
@@ -0,0 +1,1406 @@
+
+   c. Rename the `aar` extension to `zip`. So `onnxruntime-training-android-1.15.0.aar` becomes `onnxruntime-training-android-1.15.0.zip`.
+
+   d. Extract the contents of the zip file.
+
+   e. Copy the `libonnxruntime.so` shared library from the `jni\arm64-v8a` folder to your Android project under the newly created `lib` folder.
+
+   f. Copy the contents of the `headers` folder to the newly created `include\onnxruntime` folder.
+
+   g. In the `native-lib.cpp` file, include the training cxx header file.
+
+   ```cpp
+   #include "onnxruntime_training_cxx_api.h"
+   ```
+
+   h. Add `abiFilters` to the `build.gradle (Module)` file so as to select `arm64-v8a`. This setting must be added under `defaultConfig` in `build.gradle`:
+
+   ```gradle
+   ndk {
+      abiFilters 'arm64-v8a'
+   }
+   ```
+
+   i. Add the `onnxruntime` shared library to the `CMakeLists.txt` so that `cmake` can find and build against the shated library. To do this, add these lines after the `ortpersonalize` library is added in the `CMakeLists.txt`:
+
+   ```bash
+   add_library(onnxruntime SHARED IMPORTED)
+   set_target_properties(onnxruntime PROPERTIES IMPORTED_LOCATION ${CMAKE_SOURCE_DIR}/lib/libonnxruntime.so)
+   ```
+   j. Let `CMake` know where the ONNX Runtime header files can be found by adding this line right after the above two lines:
+
+   ```bash
+   target_include_directories(ortpersonalize PRIVATE ${CMAKE_SOURCE_DIR}/include/onnxruntime)
+   ```
+
+   k. Link the Android C++ project against the `onnxruntime` library by adding the `onnxruntime` library to `target_link_libraries`:
+
+   ```bash
+   target_link_libraries( # Specifies the target library.
+        ortpersonalize
+
+        # Links the target library to the log library
+        # included in the NDK.
+        ${log-lib}
+
+        onnxruntime)
+   ```
+
+   l. Build the application and wait for success to confirm that the app has included the ONNX Runtime headers and can link against the shared onnxruntime library successfully.
+
+3. Packaging the Prebuilt Training Artifacts and Dataset
+
+   a. Create a new `assets` folder inside the `app` from the left pane of the Android Studio project by right click app -> New -> Folder -> Assets Folder and place it under main.
+
+   b. Copy the above generated training artifacts to this folder.
+
+   c. Now, head over to the [`onnxruntime-training-examples`](https://github.com/microsoft/onnxruntime-training-examples/tree/master/on_device_training/mobile/android/c-cpp/data) repo and download the dataset (`images.zip`) to your machine and extract it. This dataset was modified from the orignal [`animals-10`](https://www.kaggle.com/datasets/alessiocorrado99/animals10) dataset available on Kaggle created by [Corrado Alessio](https://www.kaggle.com/alessiocorrado99). To download this dataset to your machine
```

Comment:
The ending of this instruction seems incomplete. Please review.

Reply:
Thanks. I think the last sentence is unnecessary.